string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,13 @@
+/* exported isLowerCased */
+/*
+Define a function, named 'isLowerCased', that uses one argument, named 'word'.
+  Create an if statement defining if the word is lowercased.
+    return the value true if it is lowercased.
+  return the value false if it is not.
+*/
+function isLowerCased(word) {
+  if (word === word.toLowerCase()) {
```

Comment:
This is fine but if you wanted you could reduce it to one line `return word === word.toLowerCase();` instead of using `if` `else` statements.